backend/db/initialize.py

```diff
@@ -37,10 +38,11 @@ def create_db(con: Connection):
         run_sql(con, f'USE {DB}')
     with get_db_connection(schema='') as con2:
         run_sql(con2, "CREATE TABLE IF NOT EXISTS public.manage (key text not null, value text);")
+        run_sql(con, f'CREATE SCHEMA IF NOT EXISTS {schema};')
```

Comment:
@Sigfried Added missing step to `create_db` to create Schema. I think you might have left this out for a reason though? But why?